sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -28,14 +28,22 @@ import java.util.regex.Matcher
 object CodeFormatter {
   val commentHolder = """\/\*(.+?)\*\/""".r
 
-  def format(code: CodeAndComment): String = {
+  def format(code: CodeAndComment): String = format(code, -1)
+
+  def format(code: CodeAndComment, maxLines: Int): String = {
```

Comment:
Can `maxLines` just be an optional argument rather than declare two methods? this is an internal method so its signature is OK to change.